frontend/src/views/EventDetail.vue

```diff
@@ -179,42 +185,28 @@ export default {
     h1 {{ event.title }}
     h3 {{ event.description }}
     a-card
-      a-tabs(v-if="event.owner == user._id" :default-active-key="event.owner == user._id ? '1' : '2'")
+      a-tabs(v-if="event.owner == user._id" :default-active-key="event.owner == user._id ? '2' : '1'")
         a-tab-pane(tab="Ask the speaker" key="1")
-            form(@submit.prevent="sendQuestion")
-              h2 Ask the speaker
-              a-textarea(
-                placeholder="Type your question"
-                :autoSize="{ minRows: 2, maxRows: 6 }"
-                :maxLength="280"
-                v-model="question"
-              )
-              a-input(placeholder="Your name (optional)" v-model="name" :maxLength="40")
-              a-button(type="primary" @click="sendQuestion" :loading="loading" icon="message") Send
+          ask-the-speaker-form(:loading='loading', @sendQuestion='sendQuestion($event)')
```

Comment:
we don't need to send loading and we will handle sendQuestion in AskTheSpeakerForm